queries/fusion/highlights.scm

```diff
@@ -0,0 +1,87 @@
+(comment) @comment
+
+; identifiers eel
+
+(eel_object_path
+  (eel_path_identifier) @variable)
+
+(eel_object_pair
+  key: (eel_property_name) @property)
+
+(eel_method_name) @function
```

Comment:
or `@method`? I can't tell because I don't know fusion.

Reply:
good catch - and a good question too.
I would go with function in this case, but you are right, this is a mismatch with the grammar definition.
I give you maybe some background, to start a discussion.
Eel Functions are always Helper Functions. There is nothing like object function definitions in this language specification.
Behind the scenes, there is an PHP Class, with an Implementation of this Function as Method. Maybe see the [Documentation](https://docs.neos.io/cms/manual/extending-neos-with-php-flow/custom-eel-helpers) for an example.

So in a way it is both a method (in the php implementation) and in my opinion 'only' function when using eel,